### ISSUE TITLE
machinectl support for tty password agent

### ISSUE THREAD

**Issue by risen** [systemd/systemd#13587]
- Systemd version: v241 (Debian version: 241-7~deb10u1)
- Distribution: Debian GNU/Linux 10.1 (buster)

### Problem description

As described in #6990 I use LUKS encrypted logical volumes, which I symlink in `/var/lib/machines/`. This works well. However, a small annoyance happens when the LUKS password is needed. When running machinectl the following happens:

```
~# machinectl start machine

Broadcast message from root@host (Wed 2019-09-18 10:44:18 CEST):

Password entry required for 'Please enter image passphrase:' (PID 1924).
Please enter password with the systemd-tty-ask-password-agent tool:
```
… at which point `machinectl` blocks, and I have to open a second ssh shell to the host, to execute `systemd-tty-ask-password-agent`.

### Expected behaviour

`machinectl` should immediately ask for the password, just like `systemctl` does in certain cases.

**Comment by poettering** [comment]
Fix waiting in #14037

**Comment by risen** [comment]
Nice, thanks a lot!
